care/facility/migrations/0002_new_initial.py

```
@@ -0,0 +1,47 @@
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    replaces = [
+        ('facility', '0001_initial_squashed'),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)